core/storage/user/gae_models.py

```diff
@@ -123,6 +124,15 @@ class ExplorationUserDataModel(base_models.BaseModel):
     # When the most recent rating was awarded, or None if not rated.
     rated_on = ndb.DateTimeProperty(default=None, indexed=False)
 
+    # List of uncomitted changes made by the user to the exploration.
+    change_list = ndb.JsonProperty(default=None)
```

Comment:
I am inclined to call this draft_change_list (and modify the other names accordingly) to clarify what it is for.